x-pack/legacy/plugins/siem/server/lib/detection_engine/scripts/rules/test_cases/signals_on_signals/depth_test/README.md

```diff
@@ -0,0 +1,370 @@
+This is a depth test which allows users and UI's to create "funnels" of information. You can funnel your data into smaller
+and smaller data sets using this. For example, you might have 1,000k of events but generate only 100k of signals off
+of those events. However, you then want to generate signals on top of signals that are only 10k. Likewise you might want
+signals on top of signals on top of signals to generate only 1k.
+
+```
+events from indexes might be 1,000k (no depth)
+signals -> events would be less such as 100k
+signals -> signals -> events would be even less (such as 10k)
+signals -> signals -> events would be even less (such as 1k)
+```
+
+This folder contains a rule called
+
+```sh
+query_single_id.json
+```
+
+which will write a single signal document into the signals index by searching for a single document `"query": "_id: o8G7vm8BvLT8jmu5B1-M"` . Then another rule called
+
+```sh
+signal_on_signal_depth_1.json
+```
+
+which has this key part of its query: `"query": "signal.parent.depth: 1 and _id: *"` which will only create signals
+from all signals that point directly to an event (signal -> event).
+
+Then a second rule called
+
+```sh
+signal_on_signal_depth_2.json
+```
+
+which will only create signals from all signals that point directly to another signal (signal -> signal) with this query
+
+```json
+"query": "signal.parent.depth: 2 and _id: *"
+```
+
+Setup
+---
+
+You should first get a valid `_id` from the system from the last 24 hours by running any query within timeline
+or in the system and copying its `_id`. Once you have that `_id` add it to `query_single_id.json`. For example if you have found an `_id`
+in the last 24 hours of `sQevtW8BvLT8jmu5l0TA` add it to `query_single_id.json` under the key `query` like so:
+
+```json
+"query": "_id: sQevtW8BvLT8jmu5l0TA",
+```
+
+Then get your current signal index:
+
+```json
+./get_signals_index.sh
```

Comment:
`./get_signals_index.sh` should be `/get_signal_index.sh` - no 's'.

Reply:
Thanks! Changed it